CMakeLists.txt

```diff
@@ -735,10 +735,18 @@ endif()
 include(cmake/Dependencies.cmake)
 
 # Moved this cmake set option down here because CMAKE_CUDA_COMPILER_VERSION is not avaialble until now
+if(USE_ROCM)
+  set(CONFIG_FA ON)
+elseif(USE_CUDA AND NOT CMAKE_CUDA_COMPILER_VERSION VERSION_LESS 11.6)
```

Comment:
You dropped the msvc check

Reply:
Also, we don't care about older CUDAs so...

Reply:
Fixed in f3d21d3d6dd8b830bbe2994f9e69effa8d86e14c